includes/class-amp-theme-support.php

```diff
@@ -324,10 +324,30 @@ public static function handle_xhr_request() {
 
 		if ( isset( $pagenow ) && 'wp-comments-post.php' === $pagenow ) {
 			// We don't need any data, so just send a success.
-			add_filter( 'comment_post_redirect', function() {
+			add_filter( 'comment_post_redirect', function( $url, $comment ) {
+				// Get theme support.
+				$theme_support = get_theme_support( 'amp' );
+
+				// Add the comment ID to the URL to force AMP to refresh the page.
+				$url = add_query_arg( 'comment', $comment->comment_ID, $url );
+
+				// Send redirect header if amp-live-list has been opted-out.
+				if ( empty( $theme_support['comments_live_list'] ) ) {
```

Comment:
This doesn't work because `$theme_support` is an array of the args so it needs to be looking at ` $theme_support[0]['comments_live_list']`